src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,53 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private int size = 0;
+    private K[] keys = (K[]) new Object[DEFAULT_CAPACITY];
+    private V[] values = (V[]) new Object[DEFAULT_CAPACITY];
+
     @Override
     public void put(K key, V value) {
+        for (int i = 0; i < keys.length; i++) {
+            if (isKeyRepeative(key, keys[i])) {
+                values[i] = value;
+                size++;
+                return;
+            }
+        }
+        keys[size] = key;
+        values[size] = value;
+        size++;
     }
 
     @Override
     public V get(K key) {
+        for (int i = 0; i < keys.length; i++) {
+            if (isKeyRepeative(key, keys[i])) {
+                size = getSizeWithUniqElements();
+                return values[i];
+            }
+        }
         return null;
     }
 
     @Override
     public int size() {
-        return -1;
+        return size;
+    }
+
+    private boolean isKeyRepeative(K key, K key2) {
```

Comment:
name for the method is not suitable. 
```suggestion
    private boolean isKeysEqual(K key, K key2) {
```